zipline/assets/_assets.pyx

```diff
@@ -160,14 +157,15 @@ cdef class Asset:
             'auto_close_date': self.auto_close_date,
             'exchange': self.exchange,
             'exchange_full': self.exchange_full,
+            'exchange_info': self.exchange_info,
```

Comment:
Do we want to include all three of `exchange`, `exchange_full` and `exchange_info`?

Reply:
We need the exchange_info to be be able to round trip from_dict and to_dict. I left the others for backwards compat because users may be using `to_dict()` and then reading these fields. The keys are constant so I could imagine someone assuming it is present.